src/modules/roc_netio/target_uv/roc_netio/transceiver.cpp

```diff
@@ -303,112 +300,118 @@ void Transceiver::process_tasks_() {
 }
```

Comment:
I suggest to inline task->execute() call (i.e. remove execute() method). I've spent about ten minutes to find out why we are waiting `task.done` to become true but are never setting it :)

Reply:
I do not agree. Why not to go to task->execute method and understand what it does?
I also do not like the idea of in-lining because we hide the ugliness of calling a pointer to member function inside this function.

Reply:
Because all other manipulations with Task fields are in Transceiver and are not encapsulated in Task, so I expected to see setting "done" in Transceiver and it was a surprise that this specific case is implemented somewhere else.

On the other hand, this is not very important, we can keep it as is too.

Reply:
Well, I inlined it.